src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1814,6 +1803,37 @@
       }
     },
 
+    "reportingDescriptorRelationship": {
+      "description": "Information about the relation of one reporting descriptor to another.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+
+        "descriptor": {
+          "description": "A reference to the related reporting descriptor.",
+          "$ref": "#/definitions/reportingDescriptorReference"
+        },
+
+        "kinds": {
+          "description": "A set of distinct strings that categorize the relationshship. Well-known kinds include canPrecede, canFollow, canPrecedeOrFollow, willPrecede, willFollow, superset, subset, equal, disjoint, relevant, and incomparable.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "default": [],
```

Comment:
>     "default": [], [](start = 5, length = 19)

Don't specify `default` since `kinds` is required. (Also note that the specified default violates the `minItems` constraint anyway :-).)